src/commands/buildCommands/trn.js

```diff
@@ -0,0 +1,68 @@
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+
+const { allLocalPoTrnsWithFallbacks$, localTrns$ } = require('../txCommands/lib');
+const localeCodes = require('./config/locales');
+
+const MODULES_PATH = path.resolve('./src/trns/modules/');
```

Comment:
use `paths.repoRoot` instead of a '`./`-relative' pathname this `path.resolve` - otherwise the path resolution is very confusing.